spec/yaml/swiftnav/sbp/integrity.yaml

```diff
@@ -0,0 +1,94 @@
+# Copyright (C) 2015-2021 Swift Navigation Inc.
+# Contact: https://support.swiftnav.com
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+package: swiftnav.sbp.integrity
+description: Over-the-air Integrity Messages
+stable: True
+public: False
+include:
+  - types.yaml
+  - gnss.yaml
+definitions:
+
+ - OrbitClockBound:
+    short_desc: Orbit and clock bound
+    fields:
+      - sat_id:
+          type: u8
+          desc: >
+            Satellite ID. Encoded following either RTCM DF068 (GPS), DF252 (Galileo),
+            or DF488 (BDS) according to the constellation.
+      - orbit_mean_radial:
+          type: u8
+          units: m
+          desc: Mean Radial
+      - orbit_mean_along_track:
+          type: u8
+          units: m
+          desc: Mean Along-Track
+      - orbit_mean_cross_track:
+          type: u8
+          units: m
+          desc: Mean Cross-Track
+      - orbit_std_radial:
+          type: u8
+          units: m
+          desc: Standard Deviation Radial
+      - orbit_std_along_track:
+          type: u8
+          units: m
+          desc: Standard Deviation Along-Track
+      - orbit_std_cross_track:
+          type: u8
+          units: m
+          desc: Standard Deviation Cross-Track
+      - clock_mean:
+          type: u8
+          units: m
+          desc: Clock Bound Mean
+      - clock_std:
+          type: u8
+          units: m
+          desc: Clock Bound Standard Deviation
+
+ - MSG_SSR_ORBIT_CLOCK_BOUNDS:
+    id: 1234
```

Comment:
This id is a placeholder, It must be updated to the final value before merging the PR.

Reply:
I went with ID 1502 since the bounds provided in this message are the ones for MSG_SSR_ORBIT_CLOCK (1501)

Reply:
I think the message IDs don't need to be as tightly grouped these days. Considering it is tricky figuring out the next free ID.

Reply:
Does `make mapping` show all the currently used IDs or only the public ones?
Because I was basing it on that info.